bcache/get.go

```diff
@@ -22,11 +22,35 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+
+	"github.com/prometheus/procfs/internal/fs"
 )
 
-// ReadStats retrieves bcache runtime statistics for each bcache.
-func ReadStats(sysfs string) ([]*Stats, error) {
-	matches, err := filepath.Glob(path.Join(sysfs, "fs/bcache/*-*"))
+// FS represents the pseudo-filesystem proc, which provides an interface to
+// kernel data structures.
+type FS struct {
+	sys *fs.FS
+}
+
+// DefaultSysMountPoint is the common mount point of the sys filesystem.
+const DefaultSysMountPoint = "/sys"
```

Comment:
What about having this in the `fs` package? Then we can use `fs.DefaultSysMountPoint` here.

Reply:
Good idea, saves a bit of duplication.  Done!